### PR TITLE
Documentation changes

### PR DESCRIPTION
**Pull request by sjmcdowall**
No idea why/how the example/pubspec.lock got modified but I think its harmless.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,8 +1,11 @@
 name: flutter_typeahead
+version: 0.5.2
 description: A highly customizable typeahead (autocomplete) text input field for Flutter
-version: 0.5.1
-author: AbdulRahman AlHamali <a.alhamali93@gmail.com>
 homepage: https://github.com/AbdulRahmanAlHamali/flutter_typeahead
+authors:
+  - AbdulRahman AlHamali <a.alhamali93@gmail.com>
+  - Kenneth Liang
```
Comment (AbdulRahmanAlHamali) [inline pubspec.yaml]: @KaYBlitZ Would you like to add your email here?

**Comment by sjmcdowall** [comment]
Kenneth - -how about you submitting all your fixes you made as a final PR to the develop branch?  I'd like to merge those too -- and the type fix as the next minor update .. ?? 

Then we can choose one of the PR's for the change of box location as the next mid-level bump ..

**Comment by KaYBlitZ** [comment]
Steve, I am not sure what you mean. All my previous suggestion list fixes have been merged already. It's revision 0.5.1 - Bug fixes.

**Comment by sjmcdowall** [comment]
Huh I am using a forked branch pointing to you because some extra changes and bugs were post 0.5.1?

**Comment by KaYBlitZ** [comment]
Everything in my fork has already been merged into 0.5.1. I tacked on all the changes we worked on in issue #21.

**Comment by AbdulRahmanAlHamali** [comment]
Alright then, I think we can give a shot to releasing v0.5.2?

**Comment by sjmcdowall** [comment]
Yep -- let me merge and try to pub ..

**Comment by sjmcdowall** [comment]
Well alas -- the "pub publish" didn't work as well .. 

Package has 1 warning. Upload anyway (y/n)? y
Uploading... (2.6s)
UnauthorizedAccess: Unauthorized user: sjm@hawoods.org is not allowed to upload versions to package flutter_typeahead..

That's my google login .. which dart pub uses.. doesn't appear to use GitHub auth...

**Comment by AbdulRahmanAlHamali** [comment]
@sjmcdowall I have added you as an uploader using the command:
`pub uploader add sjm@hawoords.org`
Which I learned from this article:
https://www.dartlang.org/tools/pub/cmd/pub-uploader

You should be able to upload to pub now.

@KaYBlitZ Would you share your email so that I can add you as an uploader too?

**Comment by sjmcdowall** [comment]
That worked!  

And it needs to be a Google auth email (it appears) .. doesn't mean we need to publish it (nor will we unless you want to) 

CHeers!

**Pull request merged**